### PR TITLE
Fix timestamp for update events

### PR DESCRIPTION
**Pull request by PrasadG193**
##### ISSUE TYPE

 - Bug fix Pull Request

##### SUMMARY
Fix timestamp for update events
Ignore older update events after resync

Fixes #123

### CODE REVIEW

File: pkg/events/events.go [hunk 1]
```diff
@@ -103,16 +103,40 @@ func New(object interface{}, eventType config.EventType, kind string) Event {
...
 		event.Kind = "Pod"
+		if eventType == config.UpdateEvent {
+			condLen := len(obj.Status.Conditions)
+			if condLen != 0 {
+				event.TimeStamp = obj.Status.Conditions[condLen-1].LastTransitionTime.Time
+			}
+		}
 	case *apiV1.Node:
 		event.Kind = "Node"
+		if eventType == config.UpdateEvent {
```
Comment (ssudake21) [inline pkg/events/events.go]: See this snippet being repeated across all conditions? Can we merge these somehow? See a lot of code duplication.

File: pkg/controller/controller.go [hunk 2]
```diff
@@ -47,7 +47,7 @@ func findNamespace(ns string) string {
 // RegisterInformers creates new informer controllers to watch k8s resources
 func RegisterInformers(c *config.Config) {
 	sendMessage(c, fmt.Sprintf(controllerStartMsg, c.Settings.ClusterName))
-	startTime = time.Now().Local()
+	startTime = time.Now()
```
Comment (ssudake21) [inline pkg/controller/controller.go]: Whats difference between `time.Now()` and `time.Now().Local()` ?

Comment (PrasadG193) [reply pkg/controller/controller.go]: Local() sets location with Time

File: pkg/events/events.go (hunk 1 above)
Comment (PrasadG193) [reply pkg/events/events.go]: To access resource obj, we need to typecast `runtime.Object` interface into resource object. That's what we do with switch cases. We can't access obj.Status from interface or without typecasting it

**Pull request merged**